### DISCUSSION TITLE
Scrolling behavior in 0.9.5 vs 0.10.0

### DISCUSSION THREAD

**Discussion by diocletiann** [neovim/neovim#28475] [General]
Did scrolling behavior change? 0.9.5 is much smoother in Alacritty, and I don't mind the tearing effect. In 0.10.0 there's a lot of judder. Same config.

**Comment by luukvbaal** [comment]
It did, assuming "scrolling" means pressing `Ctrl-D/U/B/F`. Please elaborate on the issue you're experiencing. And if you can reproduce from `nvim --clean`, provide the steps in an issue.

**Reply by diocletiann** [reply]
I meant holding j/k or C-e/y, not sure what the verb for that is. What I'm experiencing is the following, with the same config (on a 120hz display).

- alacritty 0.13.2, neovim 0.9.5 - mostly no judder, sometimes a "tearing" effect (similar to vsync off)

- alacritty 0.13.2, neovim master - judder in most files, no "tearing" effect

Happy to profile or provide any info you'd like. I tried a minimal.lua with `set nu` and `set nowrap` and I can see some judder but not as much.

**Reply by luukvbaal** [reply]
OK, that seems unrelated to the changes to (half-)page scrolling then. Not sure where performance might have regressed for holding down j/k.

**Reply by diocletiann** [reply]
Any ideas why the tearing effect is gone? I prefer it to the judder.

**Reply by gpanders** [reply]
You can try disabling the `'termsync'` option.

**Reply by diocletiann** [reply]
If there's a way to profile while holding j/k, I'm curious what is being called.

**Reply by diocletiann** [reply]
> You can try disabling the 'termsync' option.

Thanks! This brings back the old behavior.
